.kitchen.yml

```diff
@@ -30,3 +23,39 @@ suites:
   - name: packages
     run_list:
     - recipe[osl-php::packages]
+  - name: packages-php56
+    run_list:
+    - recipe[osl-php::packages]
+    attributes:
+      osl-php:
+        php_packages:
+          - 'devel'
+          - 'fpm'
+          - 'gd'
```

Comment:
Can we make these global attributes for all the suites and adjust the ``packages`` suite to match the tests you're doing for these tests? You may need to adjust other tests to make this work too.

Reply:
I added a commit. Is that what you had in mind?